### PR TITLE
QF-779 Exit Survey MVS Plus

### PR DESCRIPTION
**Pull request by techtoniq**

### CODE REVIEW

File: src/SFA.DAS.ApprenticeFeedback.Domain/SFA.DAS.ApprenticeFeedback.Domain.csproj [hunk 1]
```diff
@@ -4,6 +4,10 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Interfaces\IExitSurveyAttributeContext.cs" />
```
Comment (CodeWolf86) [inline src/SFA.DAS.ApprenticeFeedback.Domain/SFA.DAS.ApprenticeFeedback.Domain.csproj:8]: Is this correct? removing it from Compile?

**Review by CodeWolf86 — CHANGES_REQUESTED** [review]
Code looks good, just need to take a look at database scripts. 1 Minor comment on the csproj for domain

File: src/SFA.DAS.ApprenticeFeedback.Domain/SFA.DAS.ApprenticeFeedback.Domain.csproj (hunk 1 above)
Comment (techtoniq) [reply src/SFA.DAS.ApprenticeFeedback.Domain/SFA.DAS.ApprenticeFeedback.Domain.csproj:8]: The file was removed from the project, it should have been deleted. Sorted now.

**Review by CodeWolf86 — APPROVED** [review]
Approved code changes, now just need to sort db scripts out

**Pull request merged**